tox.ini

```diff
@@ -2,20 +2,19 @@
 requires =
     tox<4
 envlist =
-    py{37,38,39,310}-dj{32}-{unittest,pytest}
-    py{38,39,310}-dj{40}-{unittest,pytest}
-    py{38,39,310,311}-dj{41,42}-{unittest,pytest}
-    py{310,311}-dj{main}-{unittest,pytest}
+    py{38,39,310,311,312}-dj{42}-{unittest,pytest}
+    py{310,311,312}-dj{50}-{unittest,pytest}
+    #py{310,311,312}-dj{51}-{unittest,pytest}
```

Comment:
Is it a coincidence that you commented on that line?

Reply:
I don't think we should add the commented configuration for the future. it is more logical to create a Pull request after the release of this version, which will contain the necessary set of changes.